static/openapi/eds-spec.json

```diff
@@ -0,0 +1,399 @@
+          },
+          {
+            "in": "query",
+            "name": "type",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The type of events to subscribe to."
+          },
+          {
+            "in": "query",
+            "name": "filters",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "An optional query parameter to filter events."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully subscribed to stream.",
+            "headers": {
+              "X-PhotonIQ-ES": {
+                "description": "Epoch time header.",
+                "schema": {
+                  "type": "string",
+                  "description": "Epoch time in seconds."
+                }
+              },
+              "X-PhotonIQ-CustomerId": {
+                "description": "Customer ID header.",
+                "schema": {
+                  "type": "string",
+                  "description": "Customer ID that includes the tenant and fabric name."
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "error": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/es/v1/metrics": {
+      "get": {
+        "summary": "Retrieve Metrics",
+        "description": "Retrieve metrics data within a specified time range.",
+        "tags": [
+          "Event Delivery Metrics"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "start",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The start date used for querying metrics data. Must be provided in RFC3339 date-time format.",
+            "example": "YYYY-MM-DDTHH:MM:SSZ"
+          },
+          {
+            "in": "query",
+            "name": "end",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The end date used for querying metrics data. Must be provided in RFC3339 date-time format.",
+            "example": "YYYY-MM-DDTHH:MM:SSZ"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the metrics data to retrieve."
```

Comment:
Below you say "(optional)".  Here you don't.

The openapi spec has the indicator to designate it as required.  So, don't think you have to say "(optional)" since the open API UI will indicate that or have its way of conveying that.

Reply:
Removed optional text. Required is set to false.